### DISCUSSION TITLE
Limit Local Pickup option to specific shipping zones/postcodes

### DISCUSSION THREAD

**Discussion by FlorentNL** [woocommerce/woocommerce#45334] [Checkout Flow]
Hello,

Adding the option to limit the local pickup option to specific zones would be very helpful for many businesses and their CRO.

Why would a company display a big '**Local Pickup FREE**' option if the potential customer lives on the other side of the country? In my opinion, that would create unnecessary friction in the checkout flow.

The new local pickup option and integration are great. It would be a shame if some users do not use it just because they can't limit it to nearby potential customers.

Thank you all for your great work on these new cart and checkout blocks. They are very well-done!

Best regards

**Comment by senadir** [comment]
Having this would require collecting the address upfront, which is sort of the reason we introduced the new Local Pickup, which works without needing to collect an address, however we're looking at ways to filter out additional fields based on address and are likely going to revisit local pickup again.

**Reply by FlorentNL** [reply]
Hello @senadir! I was just curious if you or your team had made any progress on this matter - no pressure at all! Thanks for your time!
